lib/nerves_motd.ex

```diff
@@ -34,14 +34,14 @@ defmodule NervesMOTD do
     an empty logo (`""`) to remove it completely.
   """
   @spec print([option()]) :: :ok
-  def print(opts \\ []) do
+  def print(opts \\ [], addtl_rows_fun \\ []) do
```

Comment:
Instead of a second parameter, could you make this an option in a similar way to the `:logo` option?

For example, a user could call it like this:

```elixir
NervesMOTD.print(extra_rows: &my_row_fun/0)
```